mocap_optitrack_driver/src/mocap_optitrack_driver/mocap_optitrack_driver.cpp

```diff
@@ -97,6 +99,28 @@ void NATNET_CALLCONV process_frame_callback(sFrameOfMocapData * data, void * pUs
   static_cast<OptitrackDriverNode *>(pUserData)->process_frame(data);
 }
 
+std::chrono::nanoseconds OptitrackDriverNode::get_optitrack_system_latency(sFrameOfMocapData * data){
+  const bool bSystemLatencyAvailable = data->CameraMidExposureTimestamp != 0;
+
+  if ( bSystemLatencyAvailable ){
```

Comment:
Did you do a `colcon test`? I think the linter will fail here for style. You need an space before the '{'

Reply:
Yes, I did. 

Start testing: Jul 19 00:18 CEST
----------------------------------------------------------
End testing: Jul 19 00:18 CEST


This is my output from the test.log. It states: There where errors, but unfortunately I don't see what kind of errors.
Do you have any idea what I am missing? I updated the bracket, but I am not sure whether I am lint-compliant now or not.

Reply:
See below the comment